test/test_mirror.sh.in

```diff
@@ -83,12 +86,15 @@ echo "Launching Mirror Server"
 SERVER_ARGS="$SERVER_PORT $SERVER_VERBOSITY"
 ./mirror_server $SERVER_ARGS &
 
-./mirror_vfd
+./mirror_vfd $SERVER_PORT
 nerrors=$?
 
 echo "Stopping Mirror Server"
 ./mirror_server_stop $SERVER_PORT
 
+# Wait for background server process to exit
+wait
```

Comment:
Wait for server to finish running in the background, in case the test is running again very quickly after the previous one and the server hasn't exited yet.